### PR TITLE
Sauce Labs

### PR DESCRIPTION
**Pull request by josh**

### CODE REVIEW

**Comment by mislav** [comment]
Much boilerplate :astonished:

File: test/test.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ test('resolves promise on 500 error', function() {
   })
 })
 
-test('rejects promise for network error', function() {
```
Comment (mislav) [inline test/test.js]: I C what you did there

**Comment by josh** [comment]
@mislav any ideas for writing that one off ruby file in bash? I meant merge it into a single script.

**Comment by mislav** [comment]
I usually do:

```bash
ruby -e '
  p STDIN.read.split("/")
'
```

You can even pipe in/out of that construct. Multiline ruby script will work fine, but if you quote it with single quotes outside then inside you can only do double quotes.

**Comment by mislav** [comment]
> any ideas for writing that one off ruby file in bash? I plan merge it into a single script.

Heh, 6 months later: 07dc8ae4cc9a46ad4af35c99a8bdc0b83fbae28b

**Pull request merged**